### PR TITLE
Backported from product-docs PR #2001 (misc doc cleanup)

### PR DESCRIPTION
**Pull request by tvo318**
##### SUMMARY
This PR backports PR https://github.com/ansible/product-docs/pull/2001 to address the following issues:

[AAP-21370](https://issues.redhat.com/browse/AAP-21370) - Typo in the Automation Controller Administrator Guide
[AAP-12720](https://issues.redhat.com/browse/AAP-12720) - Remove root option in awx-manage docs
[AAP-5527](https://issues.redhat.com/browse/AAP-5527) - inventory source variables is not accurate in the controller documentation
[AAP-15184](https://issues.redhat.com/browse/AAP-15184) - Definition of "Member" role is incorrect

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - Docs

##### AWX VERSION
latest

### CODE REVIEW

File: docs/docsite/rst/administration/awx-manage.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ The *awx-manage* Utility
 .. index:: 
    single: awx-manage
 
-The ``awx-manage`` utility is used to access detailed internal information of AWX. Commands for ``awx-manage`` should run as the ``awx`` or ``root`` user.
+The ``awx-manage`` utility is used to access detailed internal information of AWX. Commands for ``awx-manage`` should only run as the ``awx`` user.
```
Comment (oraNod) [inline docs/docsite/rst/administration/awx-manage.rst]: This is just a nit. I realize it's a backport so a minor change might not be worth the hassle. That said I always try to be careful about the use of the word "only". In this instance I would prefer to say:

`should run as the awx user only` - the position here makes it clear that "only" qualifies the user that runs the command.

`should only run` - on the other hand, the position here seems to qualify the verb "run" - as if to say you should only run the command, not perform some other action.

again this is a minor style thing and the meaning is clear. just thought I'd point that out for reference.

**Review by oraNod — APPROVED** [review]
pointed out a minor style nit but other than that LGTM. thanks @tvo318

File: docs/docsite/rst/administration/awx-manage.rst (hunk 1 above)
Comment (tvo318) [reply docs/docsite/rst/administration/awx-manage.rst]: Good point!

Comment (tvo318) [inline docs/docsite/rst/administration/awx-manage.rst]:
```suggestion
The ``awx-manage`` utility is used to access detailed internal information of AWX. Commands for ``awx-manage`` should run as the ``awx`` user only.
```

**Pull request merged**